config/routes.rb

```diff
@@ -8,8 +8,8 @@
   # Defines the root path route ("/")
   # root "posts#index"
   get "/api/v1/items/:id/merchant", to: "api/v1/items#merchant"
-  get "/api/v1/merchants/find", to: "api/v1/find_merchants#index"
-  get "/api/v1/items/find_all", to: "api/v1/find_items#index"
+  get "/api/v1/merchants/find", to: "api/v1/items/item_merchants#index"
+  get "/api/v1/items/find_all", to: "api/v1/merchants/merchant_items#index"
  
```

Comment:
Looking at these now, I think we can refactor these and drop some of their file path if we nest them in the namespace.
e.g.
namespace :api do
  namespace v1 do
    get "/items/find_all", to: "merchants/merchant_items#index"